refresh.template.py

```diff
@@ -147,6 +148,17 @@ def _get_headers_msvc(compile_args: typing.List[str], source_path: str):
         # End:   template filled by Bazel
     ))
 
+    # Write header_cmd to a temporary file, so we can use it as a parameter to cl.exe,
+    # because Windows cmd has a limitation of 8KB of command line length. So here we make a threshold of len(compile_args) < 8000
+    WIN_CMD_LIMITS = 8000
+    if len('\n'.join(header_cmd)) >= WIN_CMD_LIMITS:
+        fd, temp_params = tempfile.mkstemp(text=True)
```

Comment:
Per your question in the issue: Let's clean up the temp file when we're done, because there could be a lot of these! How about using one of the other helpers that auto deletes w/ a context manager, like tempfile.NamedTemporaryFile?

Reply:
I intend to do this, but just noticed Bazel leave quite a lot temp files there :-(
Will fix it.

Reply:
Bazel's are caches, though, hopefully, right? If they're just accumulating piles of temp files on Windows, it might be worth filing a quick issue with them.